UI/BindingSources/CSHARP/Binding/Binding.csproj

```diff
@@ -0,0 +1,45 @@
+<Project Sdk="Uno.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>
+      net8.0-android;
+      net8.0-ios;
+      net8.0-maccatalyst;
+      net8.0-windows10.0.19041;
+      net8.0-desktop;
+      net8.0-browserwasm;
+    </TargetFrameworks>
```

Comment:
Roslyn has some issues... go ahead and collapse this to a single line. see https://github.com/unoplatform/uno.templates/issues/654